### PR TITLE
Remove redundant package aliases from generated imports

### PR DESCRIPTION
**Pull request by hasLeland**
Since the packages named 'svc' and 'middlewares' are actually in directories with those names now, we no longer need to alias them to those names.

- [x] Remove redundant 'svc' package alias within:
    - [X] `cmd/NAME-server/main.gotemplate`
    - [X] `svc/client/http/client.gotemplate`
    - [X] `svc/client/grpc/client.gotemplate`
    - [X] `middlewares/endpoints.gotemplate`
- [X] Remove redundant 'middlewares' aliases within:
    - [X] `cmd/NAME-server/main.gotemplate`
- [X] Remove redundant 'handler' aliases in the:
    - [X] `cmd/NAME-server/main.gotemplate`, also change references to 'handlers'

### CODE REVIEW

File: gengokit/template/NAME-service/cmd/NAME-server/main.gotemplate [hunk 1]
```diff
@@ -21,9 +21,9 @@ import (
 
 	// This Service
 	handler "{{.ImportPath -}} /handlers"
```
Comment (adamryman) [inline gengokit/template/NAME-service/cmd/NAME-server/main.gotemplate]: `handler` should be removed as well, and all places that currently call `handler` should be updated to `handlers`. Could you roll that into this pull?

Comment (adamryman) [reply gengokit/template/NAME-service/cmd/NAME-server/main.gotemplate]: Also anywhere else `handler` is used as well.

Comment (hasLeland) [reply gengokit/template/NAME-service/cmd/NAME-server/main.gotemplate]: Done!

**Comment by adamryman** [comment]
How about removing `clientHandler` in `gengokit/template/NAME-service/cmd/NAME/main.gotemplate` and updating all calls to it `handlers`? Thanks

The package should also be updated to `package handlers` rather than `package clienthandler`

**Review by adamryman — APPROVED** [review]

**Pull request merged**